tests/e2e/default/targets/target-aggregates.wdio-spec.js

```diff
@@ -278,13 +279,71 @@ describe('Target aggregates', () => {
         expect(await contactsPage.getContactInfoName()).to.equal('Prometheus');
         // assert that the activity card exists and has the right fields.
         expect(await contactsPage.getContactCardTitle()).to.equal('Activity this month');
-        await helperFunctions.validateCardFields(['yesterday Prometheus', '18', '15%']);
+        await helperFunctions.validateCardFields(['yesterday Prometheus', moment().format('YYYY-MM'), '18', '15%']);
 
         await browser.back();
         const secondTargetItem =
           await (await targetAggregatesPage.getTargetItem(expectedTargets[1], CURRENT_PERIOD, districtHospital1.name));
         await helperFunctions.assertTitle(secondTargetItem.title, expectedTargets[1].title);
       });
+
+      it('should display targets of current user on home place', async () => {
```

Comment:
Have you considered testing this with an offline user instead?

Reply:
it should be the same for the offline user. I just used one user that was available.